config

```diff
@@ -39,3 +40,10 @@ PERSISTENT_KEEPALIVES=25
 
 # Allowed IP's (CIDR) on wireguard; for boot this should be the peer (server).
 ALLOWED_IPS=172.31.255.254/32
+#ALLOWED_IPS_6=fc10:4:0::0/128
+
+#Set this if you need to force a different MTU than wireguard automatically sets
+#WG_MTU=1200
```

Comment:
Explicitly set this to the default WG value; users can determine if they need to shorten it for use. The default value is 1420 (https://github.com/WireGuard/wireguard-tools/blob/13f4ac4cb74b5a833fa7f825ba785b1e5774e84f/src/wg-quick/linux.bash#L141).

Use active voice for comment to make it more succinct; and keep or wrap at 80 characters for readability. Something like `Wireguard MTU size.`